src/Lecture2.hs

```diff
@@ -194,7 +294,11 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge [] right = right
+merge left [] = left
+merge (x : xs) (y : ys)
+    | x > y = [y] ++ merge (x : xs) ys
```

Comment:
You can use the cons operator to add the element in the beginning. It will be slightly more efficient than adding a list with `++`
```suggestion
    | x > y = y : merge (x : xs) ys
```

Reply:
Oh, yeah! Somehow it didn't click me to use cons operator in this case and other than in (x : xs)!